pygotham/frontend/about.py

```diff
@@ -1,28 +1,35 @@
 """About PyGotham."""
 
-from flask import Blueprint
+from collections import defaultdict
 
-from pygotham.frontend import direct_to_template
+from flask import Blueprint, render_template, url_for
+
+from pygotham.events import get_current
+from pygotham.models import AboutPage
 
 __all__ = ('blueprint',)
 
 blueprint = Blueprint('about', __name__, url_prefix='/about')
 
-direct_to_template(
-    blueprint,
-    '/code-of-conduct',
-    template='about/code-of-conduct.html',
-    endpoint='code_of_conduct',
-)
-direct_to_template(
-    blueprint,
-    '/privacy-policy',
-    template='about/privacy-policy.html',
-    endpoint='privacy_policy',
-)
-direct_to_template(
-    blueprint,
-    '/faq',
-    template='about/faq.html',
-    endpoint='faq',
-)
+
+def get_nav_links():
+    """Generates all about page titles and urls for use in the navbar."""
+    links = defaultdict(dict)
```

Comment:
This is trickier than I thought at first glance. Because results from `get_nav_links` get called with `dict.update` two levels deep (navbar section and link overrides), mixing ordered and unordered content can be weird. At this time, the PR will currently merge and then alphabetize each section. Going forward, we can play with other ideas (such as pushing all overridden content above the rest and letting that get automatically sorted).

In the interest of time, I'd like to punt on the custom ordering issue in favor of alphabetized links until we have a better idea on how to handle this merge logic.